samples/subsys/nvs/CMakeLists.txt

```diff
@@ -1,6 +1,13 @@
 # SPDX-License-Identifier: Apache-2.0
 
 cmake_minimum_required(VERSION 3.13.1)
+
+macro(app_set_runner_args)
+  board_runner_args(dfu-util "--dfuse-modifiers=force:mass-erase")
+  board_runner_args(pyocd "--erase")
+  board_runner_args(nrfjprog "--erase")
```

Comment:
I think that correct parameter for `nrfjprog`, if you really want to erase entire flash is `--eraseall`

```
-e  --eraseall              Erases all user available program flash memory and
                             the UICR page. Can be combined with the
                             --qspieraseall operation.
                             Limitations:
                             For nRF51 devices, if the device came from Nordic
                             with a pre-programmed SoftDevice, only the user
                             available code flash and UICR will be erased.
```

Reply:
This is a west flash argument, not an nrfjprog argument. It is correct.